paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/definitions/CardDefinition.kt

```diff
@@ -17,6 +18,10 @@ import com.stripe.android.ui.core.elements.SharedDataSpec
 internal object CardDefinition : PaymentMethodDefinition {
     override val type: PaymentMethod.Type = PaymentMethod.Type.Card
 
+    override val supportedAsSavedPaymentMethod: Boolean = true
+
+    override fun addRequirement(hasIntentToSetup: Boolean): Set<AddPaymentMethodRequirement> = setOf()
```

Comment:
This is an inline function which maps to `emptySet`. I chose to do setOf for all of these to stay consistent, and make it easy to add requirements in the future, but don't feel strongly about the decision.